### ISSUE TITLE
Request for a small test dataset

### ISSUE THREAD

**Issue by stephanflemming** [FredHutch/SEACR#29]
Hi,

I am working on a Galaxy wrapper of SEACR. Can you recommend a small dataset for the internal tests of my implementation?

You can find the PR here https://github.com/galaxyproject/tools-iuc/pull/2852

Thanks in advance,
Stephan

**Comment by stephanflemming** [comment]
Wrapper is finished. I am still lacking of test data for a control bedgraph.

The wrapper can be tested on usegalaxy.eu [![Galaxy tool](https://img.shields.io/static/v1?label=usegalaxy&message=eu&color=blue)](https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/seacr/seacr/)

**Comment by mpmeers** [comment]
Hi Stephan,

Apologies for the delay in responding. I've created a directory on the main page called "Testfiles" that contains two bedgraphs corresponding to CTCF and IgG CUT&RUN data from human definitive endoderm, containing only the first 100 Mb of chr1. I've also uploaded the expected output files for normalized stringent mode and normalized relaxed mode, which each took around 8 seconds for me to generate on our computing cluster. I've included a more detailed description of how I generated them in the file "Test_README.md" under the Testfiles directory.

Of note, we have implemented a web server ([https://seacr.fredhutch.org](https://seacr.fredhutch.org)) for running SEACR on a web interface, though obviously a Galaxy wrapper will enable better integration with other Galaxy pipelines. Thank you for your efforts on this, and let me know if the test files work for you.

Mike
